hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -503,18 +509,21 @@ public void testBlockDeletion() throws Exception {
   @Test
   public void testWithUnrecordedBlocks() throws Exception {
     // Skip schemaV1, when markBlocksForDeletionSchemaV1, the unrecorded blocks
-    // from received TNXs will be skipped to delete and will not be added into
-    // the NumPendingDeletionBlocks
+    // from received TNXs will be deleted, not in BlockDeletingService
     if (Objects.equals(schemaVersion, SCHEMA_V1)) {
```

Comment:
This if statement can be replaced by `Assume.assumeFalse(Objects.equals(schemaVersion, SCHEMA_V1))`.

Reply:
The logic is same, but the test will be marked as "skipped" instead of "pass".

Reply:
Thx for the explanation! Updated